lib/models/download.js

```diff
@@ -5,7 +5,33 @@ import scopeChecks from 'lib/models/plugins/scopeChecks';
 import * as scopes from 'lib/constants/scopes';
 import uploadSchema from 'lib/models/uploadSchema';
 import addCRUDFunctions from 'lib/models/plugins/addCRUDFunctions';
+import { factory } from 'lib/services/files/storage';
 import _ from 'lodash';
+import { SITE_SETTINGS_ID } from 'lib/constants/siteSettings';
+import logger from 'lib/logger';
+import SiteSettings from './siteSettings';
+import Organisation from './organisation';
+import moment from 'moment';
+
+const getExpireTTL = async (download) => {
```

Comment:
Probably would have called this `getOrgTTL` and just passed the organisation ID through instead of the entire download so it's quicker to understand.

Reply:
https://github.com/LearningLocker/learninglocker/pull/1215/commits/9d36d42705a438fdbe94fa972e7840a53a281dfa fixed